README.md

```diff
@@ -62,6 +62,10 @@ Branch pr/999 set up to track remote branch pr/999 from origin.
 Switched to a new branch 'pr/999'
 ```
 
+## Merging a PR:
+
+Each repository will have its own reviewers and CI requirements. Small PRs should be squashed then merged with a single "Merge pull request #XYZ from [branch]: [title]" commit. Larger PRs can include multiple commits, but should squash related commits for brevity, prefix each commit with the PR number, and include the same single last merge commit with the PR title.
```

Comment:
Meh that commit message is a bit verbose - how about just:
```
Merge PR [Title]
```

Reply:
I think we want to include the number since the other commits include the number. Removed the branch.